src/main/java/no/unit/nva/doi/transformer/DataciteResponseConverter.java

```diff
@@ -132,15 +274,15 @@ protected String extractMainTitle(DataciteResponse response) {
 
     protected List<Contributor> toContributors(List<DataciteCreator> creators) {
         return IntStream.range(0, creators.size()).mapToObj(i -> toCreator(creators.get(i), i + 1)).collect(
-            Collectors.toList());
+                Collectors.toList());
     }
 
     protected Contributor toCreator(DataciteCreator dataciteCreator, Integer sequence) {
         try {
             return new Contributor.Builder().withIdentity(
-                new Identity.Builder().withName(toName(dataciteCreator)).withNameType(
-                    NameType.lookup(dataciteCreator.getNameType())).build()).withAffiliations(
-                toAffilitations()).withSequence(sequence).build();
+                    new Identity.Builder().withName(toName(dataciteCreator)).withNameType(
```

Comment:
Refactor and lint for better readability.

```suggestion
                    protected Contributor toCreator(DataciteCreator dataciteCreator, Integer sequence) {
        try {
            return new Contributor.Builder()
                .withIdentity(createCreatorIdentity(dataciteCreator))
                .withAffiliations(toAffilitations())
                .withSequence(sequence)
                .build();
        } catch (MalformedContributorException e) {
            return null;
        }
    }

    private Identity createCreatorIdentity(DataciteCreator dataciteCreator) {
        return new Identity.Builder().withName(toName(dataciteCreator))
            .withNameType(NameType.lookup(dataciteCreator.getNameType())).build();
    }
```